cfme/fixtures/ansible_fixtures.py

```diff
@@ -101,6 +102,26 @@ def ansible_service_catalog(appliance, ansible_catalog_item, ansible_catalog):
     return service_catalog_
 
 
+@pytest.fixture
+def ansible_service_request(appliance, ansible_catalog_item):
+    request_descr = "Provisioning Service [{0}] from [{0}]".format(ansible_catalog_item.name)
+    service_request_ = appliance.collections.requests.instantiate(description=request_descr)
```

Comment:
Rename the variable and remove underscore at the end
service_request = appliance.collections.requests.instantiate(description=request_descr)